### ISSUE TITLE
feat: use voting classifier instead of individual

### ISSUE THREAD

**Issue by saidsef** [saidsef/ml-classifier#161]
Problem Statement:

We use multiple stand-alone classifiers then we have to train / load them individually before we can test and inference.

This is time consuming and as always one or more of the engines fails to perform when source data is updated.  We need a way to combine and predict probability of the inference.

Proposed Solution:

scikit-learn has what is called `VotingClassifier` that can be provided with list of estimators, the limitation of this is that the estimator must support `predict_proba` - else it will throw an error.

This has the benefit of soft Voting / Majority Rule, the idea is to combine conceptually different machine learning classifiers and use a majority vote or the average predicted probabilities (soft vote) to predict the class labels.

Action Items:
- Update Notepad and Classifier class to accommodate `VotingClassifier` and required libraries
- Update training data and test data
- Update CI/CD pipeline to add news combined model
